xpk.py

```diff
@@ -1392,28 +1838,51 @@ def run_gke_node_pool_create_command(args, system) -> int:
 
   commands = []
   task_names = []
+
+  xpk_print(
+      f'Creating {args.num_slices} node pool or pools of {system.device_type}\n'
+      f'Underlyingly, we assume that means: {system}'
+  )
+
+  if system.device_type == h100_device_type:
+    desired_node_pool_names = [f'{args.cluster}-np-0']
+  else:
+    desired_node_pool_names = [
+      f'{args.cluster}-np-{slice_num}' for slice_num in range(args.num_slices)
+    ]
+
   for node_pool_name in desired_node_pool_names:
     if node_pool_name in existing_node_pool_names:
       continue
     command = (
         'gcloud beta container node-pools create'
-        f' {node_pool_name} --node-version={args.gke_version}'
+        f' {node_pool_name}'
+        f' --region={zone_to_region(args.zone)}'
         f' --cluster={args.cluster}'
         f' --project={args.project} --node-locations={args.zone}'
-        f' --region={zone_to_region(args.zone)}'
-        f' --num-nodes={system.vms_per_slice}'
         f' --machine-type={system.gce_machine_type}'
         f' --host-maintenance-interval={args.host_maintenance_interval}'
         f' {capacity_args}'
-        ' --scopes=storage-full,gke-default'
-        ' --enable-gvnic --max-pods-per-node 15'
+        ' --enable-gvnic'
     )
     if system.accelerator_type == AcceleratorType['TPU']:
-      command += (' --placement-type=COMPACT ')
+      command += (f' --node-version={args.gke_version}')
+      command += (f' --num-nodes={system.vms_per_slice}')
+      command += (' --placement-type=COMPACT  --max-pods-per-node 15')
+      command += (' --scopes=storage-full,gke-default')
       command += (f' --tpu-topology={system.topology}')
       command += (f' {args.custom_tpu_nodepool_arguments}')
-    elif system.accelerator_type == AcceleratorType['GPU']:
-      command += (' --placement-type=COMPACT ')
+    elif system.device_type == h100_device_type:
+      command += (f' --num-nodes={args.num_slices}')
+      command += (f' --accelerator type={system.gke_accelerator},count={str(system.chips_per_vm)}'
+        f' --additional-node-network network={args.cluster}-net-1,subnetwork={args.cluster}-sub-1'
+        f' --additional-node-network network={args.cluster}-net-2,subnetwork={args.cluster}-sub-2'
+        f' --additional-node-network network={args.cluster}-net-3,subnetwork={args.cluster}-sub-3'
+        f' --additional-node-network network={args.cluster}-net-4,subnetwork={args.cluster}-sub-4'
+        ' --no-enable-autoupgrade  --scopes="https://www.googleapis.com/auth/cloud-platform"'
+        )
+    else: # other gpu types
```

Comment:
We also have CPU Types which would fall into this else statement. Can you add an `elif system.accelerator_type == AcceleratorType['GPU']` gpu case? (if needed)

Or add a comment saying other GPU and CPU Types.